server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -68,5 +76,20 @@ export function create(
                 201);
     });
 
+    router.delete("/repos/:ignored?/:tenantId/git/summaries",
+    throttle(throttler, winston, commonThrottleOptions),
```

Comment:
nit: add another tab here and below to make the spacing consistent with the other `router` statements above